cernopendata/modules/fixtures/data/docs/cern-open-data-release-2024/cern-open-data-release-2024.md

```diff
@@ -0,0 +1,44 @@
+A new version of the CERN Open Data Repository has been deployed in January 2024. The changes were deployed first in the [CERN Open Data QA instance](https://opendata-qa.cern.ch), and then propagated to the production instance.
+
+The list of changes include:
+
+* Change of technology, from Angular to ReactJS and Bootstrap[3,4] to Semantic UI for the web interface
+* Update the dependencies on the invenio framework modules
```

Comment:
This news item is also formulated in a very internal manner, the eople would not know what this change brings, and would wonder "what this brings in for me?". I would leave it out (similarly to the docker  news) or else explain what the Invenio module upgrade brings.  (E.g. "improving exposure of records by sitemaps" to come up with a simple example.)

Reply:
Using the same logic, the xrootd part should also be out. That would be a bit too much. What about combining the three of them into a single entry of update dependencies?  People do know that dependencies tend to be get obsolete, and htat it is good to be to the recent versions